pkg/apis/internal.acorn.io/v1/event.go

```diff
@@ -97,15 +97,10 @@ func (t *MicroTime) UnmarshalJSON(b []byte) error {
 
 const (
 	// EventSeverityInfo indicates an event describes a system operating "as expected".
-	// It is the lowest severity level.
 	EventSeverityInfo EventSeverity = "info"
 
-	// EventSeverityWarn indicates an event describes a recoverable error.
-	EventSeverityWarn EventSeverity = "warn"
-
-	// EventSeverityCritical indicates an event describes an unrecoverable error.
-	// It is the highest severity level.
-	EventSeverityCritical EventSeverity = "critical"
+	// EventSeverityError indicates an event describes an error.
+	EventSeverityError EventSeverity = "warn"
```

Comment:
Just checking if this was intentional or not since it confused me at first glance.
```suggestion
	EventSeverityError EventSeverity = "error"
```